prime-dotnet-webapi/Services/MongoServices/LocationService.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using Prime.Models;
+using Prime.Services;
+
+namespace Prime.Services
+{
+    public class LocationService : ILocationService
```

Comment:
I am a little worried about the "POC" implementation coming into our code like this. It's not live or being used. There is a chance we move this into a different parent subfolder (not in prime-dotnet-webapi) as well. So it's a little early to have these implementation items here. I like that we took the POC to the point of getting data to work via our services.. but maybe that could stay behind in a branch instead.

Reply:
Do you want me to remove the insert of the EnrolleeProfileVersion into the Mongo database as well?